common/src/main/java/org/jboss/hal/testsuite/fragment/config/resourceadapters/ResourceAdapterPropertyWizard.java

```diff
@@ -0,0 +1,29 @@
+package org.jboss.hal.testsuite.fragment.config.resourceadapters;
+
+import org.jboss.hal.testsuite.fragment.shared.modal.WizardWindow;
+
+/**
+ * Class representing a dialog window for creating new resource adapter property
+ */
+public class ResourceAdapterPropertyWizard  extends WizardWindow {
+    private static final String NAME = "name";
```

Comment:
question now is which one is more generally used, the one with "name" or the one with "key"?

Reply:
The one with the "key"